### PR TITLE
Copy the PAQ 10.18 release notes into the C8Y 10.17 release notes

### PR DESCRIPTION
**Pull request by kailol-sag**
The Streaming Analytics release notes in the 10-17-0 and 10-18-0 bundles are now identical.
This applies to both the individual release notes for Streaming Analytics and the Important announcements.
The only exception is the version number in the URL, which always points to the doc for the corresponding version.

### CODE REVIEW

File: content/release-10-17-0/announcements-10-17-0.md [hunk 1]
```diff
@@ -185,41 +185,53 @@ As announced with [release 10.15](/release-10-15-0/announcements-10-15-0) and [r
...
+<!-- Had to rewrite the "Planned" and move to "Implemented".
+However, this is also mentioned in the "streaming-analytics-10-17-0" bundle, but without mentioning the German doc.
+So where do we want to have this info? -->
+The Analytics Builder documentation has been integrated into the Cumulocity IoT documentation.
+Analytics Builder developers can now search and reference the Analytics Builder documentation within the Cumulocity IoT documentation.
 This makes it easier for new users to get started and for advanced users to use a single documentation interface.
-The Analytics Builder documentation will then no longer be available as a separate webhelp.
-The German version of the Analytics Builder documentation, which is also available as a separate webhelp until release 10.17, is deprecated.
-We will focus on our high-quality, up-to-date English user documentation.
+The Analytics Builder documentation is no longer be available as a separate webhelp.
```
Comment (BeateRixen) [inline content/release-10-17-0/announcements-10-17-0.md:214]:
```suggestion
The Analytics Builder documentation is no longer available as a separate webhelp.
```

File: content/release-10-17-0/announcements-10-17-0.md [hunk 2]
```diff
@@ -185,41 +185,53 @@ As announced with [release 10.15](/release-10-15-0/announcements-10-15-0) and [r
...
+##### New sample for calling another microservice
+<!-- from 10.18 release notes -->
+As announced with [release 10.16](/release-10-16-0/announcements-10-16-0), the EPL Apps sample "Call Zementis microservice" sample has been removed.
+It has been replaced by a new "Call another microservice" sample which now uses the `/health` endpoint of an Apama-ctrl microservice.
+See also [Connecting Apama to other microservices](https://cumulocity.com/guides/10.17.0/streaming-analytics/epl-apps/#microservices) in the *Streaming Analytics guide*.
 
-In a future release, we are going to integrate the Analytics Builder documentation into the Cumulocity IoT documentation.
-Analytics Builder developers will then be able to search and reference the Analytics Builder documentation within the Cumulocity IoT documentation.
+##### Documentation
+<!-- Had to rewrite the "Planned" and move to "Implemented".
```
Comment (BeateRixen) [inline content/release-10-17-0/announcements-10-17-0.md:208]: This is actually not an "Important announcement", in the way we usually use it, i.e. for breaking changes or any othe r changes which require customer action. Although they have to change links, bookmarks. The deprecation/removal of the German version could go here but perhaps we move the announcement of the integration of the Analytics Builder guide to the Whats New section?

Comment (kailol-sag) [reply content/release-10-17-0/announcements-10-17-0.md]: I kept the info on the German version here. And I've shortened the info on the English version. What do you think, @BeateRixen ?

Comment (BeateRixen) [reply content/release-10-17-0/announcements-10-17-0.md]: Looks good to me. But the sentences on the German version should also go under "Implemented", shouldn´t they?

Comment (kailol-sag) [reply content/release-10-17-0/announcements-10-17-0.md]: The German sentences are already under "Implemented". Don't let the diff confuse you :-)

File: content/release-10-18-0/streaming-analytics-10-18-0-bundle/10_18_0.md [hunk 3]
```diff
@@ -4,6 +4,12 @@ title: Release 10.18.0
 layout: redirect
 ---
 
+<!-- The following has been copied to the 10.17 release notes.
+TODO:
```
Comment (skom-sag) [inline content/release-10-18-0/streaming-analytics-10-18-0-bundle/10_18_0.md:8]: ok, lets go ahead with that plan

File: content/release-10-18-0/announcements-10-18-0.md [hunk 4]
```diff
@@ -152,6 +152,10 @@ As announced with [release 10.17](/release-10-17-0/announcements-10-17-0), with
 
 
 ### Streaming Analytics
+<!-- The following has been copied to the 10.17 release notes.
+TODO: Make this identical to the 10.17 release notes, and sync any future changes.
```
Comment (skom-sag) [inline content/release-10-18-0/announcements-10-18-0.md:156]: ok

File: content/release-10-18-0/streaming-analytics-10-18-0-bundle/10_18_0_1.md [hunk 5]
```diff
@@ -3,6 +3,9 @@ weight: 39
 title: Release 10.18.0.1
 layout: redirect
 ---
+<!-- Question:
+What to do about this separate topic for the 10.18.0.1 fix? Remove it?
```
[the remark this answers is not in the document]
Comment (kailol-sag) [reply content/release-10-18-0/streaming-analytics-10-18-0-bundle/10_18_0_1.md:7]: Does that mean that I can delete this topic?
The info from that topic is now in this file: /release-10-17-0/streaming-analytics-10-17-0-bundle/10_17_0.md (which contains 10.18 info despite the 10_17_0 in the file name).

[the remark this answers is not in the document]
Comment (skom-sag) [reply content/release-10-18-0/streaming-analytics-10-18-0-bundle/10_18_0_1.md:7]: delete the topic

[the remark this answers is not in the document]
Comment (kailol-sag) [reply content/release-10-18-0/streaming-analytics-10-18-0-bundle/10_18_0_1.md]: Done, deleted.

File: content/release-10-18-0/announcements-10-18-0.md (hunk 4 above)
Comment (kailol-sag) [reply content/release-10-18-0/announcements-10-18-0.md:156]: Done.

**Review by skom-sag — APPROVED** [review]

**Review by BeateRixen — APPROVED** [review]

**Pull request merged**